README.md

```diff
@@ -61,23 +61,25 @@ ReactJS based Presentation Library
 
 ## Getting Started
 
-The new best way to get started is by running `create-react-app my-presentation --scripts-version spectacle-scripts`. This will use `create-react-app` to create almost everything you need. This however, doesn't include publish scripts, and ejecting is required for fancy stuff.
+There are three ways to get started on your presentation. The first way is to run the following commend in your terminal.
 
-The second best way to get started is by using the [Spectacle Boilerplate](https://github.com/FormidableLabs/spectacle-boilerplate).
+`npx create-react-app my-presentation --scripts-version spectacle-scripts`
 
-Alternatively, you can `npm install spectacle` and write your own build configurations. We also provide full UMD builds (with a `Spectacle` global variable) of the library at `dist/spectacle.js` and `dist/spectacle.min.js` for more general use cases. You could, for example, include the library via a script tag with: `https://unpkg.com/spectacle@VERSION/dist/spectacle.min.js`.
+The second way to get started is by using the [Spectacle Boilerplate](https://github.com/FormidableLabs/spectacle-boilerplate).
 
-Note that we have webpack externals for `react`, `react-dom`, and `prop-types`, so you will need to provide them in your upstream build or something like linking in via `script` tags in your HTML page for all three libraries. This comports with our project dependencies which place these three libraries in `peerDependencies`.
+Both of these ways will give you everything you'll need to get started, including a sample presentation in the `presentation.js` file. You can changed the props and tags as needed for your presentation or remove the `Deck` tag and all its content to start from scratch. From here you can go to [Development](#development) to get started.
 
-But really, it is SO much easier to just use the boilerplate. Trust me.
+The third way to get started, you can run `npm install spectacle` in your terminal and write your own build configurations. We also provide full UMD builds (with a `Spectacle` global variable) of the library at `dist/spectacle.js` and `dist/spectacle.min.js` for more general use cases. You could, for example, include the library via a script tag with: `https://unpkg.com/spectacle@VERSION/dist/spectacle.min.js`.
```

Comment:
`The third way to get started is by running `npm install spectacle` in your terminal and writing your own build configurations.`